web/src/frontend/src/digisos/skjema/DigisosSkjemaSteg.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from "react";
```

Comment:
Ny komponent som setter opp skjemaet for søknaden. Hvert steg bruker denne som wrapper.